src/data/meta.ts

```diff
@@ -195,7 +195,7 @@ export class MetaDataManager {
     return this.getSensor({ id: 'jhu-csse', signal: 'confirmed_7dav_incidence_prop' });
   }
   getDefaultDeathSignal(): Sensor | null {
-    return this.getSensor({ id: 'jhu-csse', signal: 'deaths_7dav_incidence_prop' });
+    return this.getSensor({ id: 'nchs-mortality', signal: 'deaths_covid_incidence_prop' });
   }
```

Comment:
If we are getting rid of `getDefaultCasesSignal()` above, we should get rid of this one too.  They dont appear to still be used anywhere.